### ISSUE TITLE
Not work with FormData

### ISSUE THREAD

**Issue by MeloJR** [developit/redaxios#28]
When you pass `FormData` objects as the data parameter the library sends the request as `application/json`.

The problem is here:
```
		if (data && typeof data === 'object') {
			data = JSON.stringify(data);
			customHeaders['Content-Type'] = 'application/json';
		}
```
as `typeof new FormData() === 'object'`.

**Comment by developit** [comment]
Good catch. Happy to accept a PR, this will likely require inspecting the constructor.

**Comment by crissto** [comment]
Hi!
would the solution to this be, check if data is FormData and set a different Content-Type? If it's that I can fix it and send a PR

**Comment by fabricionaweb** [comment]
@crissto I guess we can just check `!(data instanceof FormData)`
But not sure if that has side-effects... hope @developit knows more than me about it :D

**Comment by developit** [comment]
@fabricionaweb yup, that's likely going to be the smallest check. Another option would be checking `if (typeof data.append=='function'){}`, but the size win likely isn't worth the false positive chance.

**Comment by developit** [comment]
Fix published in [version 0.3.0](https://github.com/developit/redaxios/releases/tag/0.3.0).
